extensions/integration-tests/src/utils.ts

```diff
@@ -105,13 +104,33 @@ export async function deleteDB(dbName: string, ownerUri: string): Promise<void>
 			SELECT ERROR_MESSAGE() AS ErrorMessage;
 		END CATCH`;
 
-	await queryProvider.runQueryAndReturn(ownerUri, query);
+	ownerUri = await ensureServerConnected(server, ownerUri);
+	let dbDeleteResult = await this.runQuery(query, ownerUri);
+	assert(dbDeleteResult.columnInfo[0].columnName !== 'ErrorMessage', 'DB deletion threw error');
 }
 
+export async function ensureServerConnected(server: TestServerProfile, ownerUri: string): Promise<string> {
+	let connection = await azdata.connection.getConnection(ownerUri);
+	if (isNullOrUndefined(connection)) {
```

Comment:
Would suggest adding a comment explaining why you're doing this.

Reply:
there is comment in line 115